docs/docusaurus.config.js

```diff
@@ -2,211 +2,214 @@
-            dropdownItemsAfter: [
-              {
-                href: "http://botpress-docs.s3-website-us-east-1.amazonaws.com/docs/installation/",
-                label: "v12.26.7",
-              },
-              {
-                href: "http://botpress-docs.s3-website-us-east-1.amazonaws.com/docs/11.9.6/installation/",
-                label: "v11.9.6",
-              },
-              {
-                href: "http://botpress-docs.s3-website-us-east-1.amazonaws.com/versions",
-                label: "(Older docs) 11.0 to v12.26",
-              },
-            ],
-            dropdownActiveClassDisabled: true,
-          },
-          {
-            type: "search",
-            position: "right",
-          },
-        ],
-      },
-      footer: {
-        style: "light",
-        links: [
-          {
-            title: "Docs",
-            items: [
-              {
-                label: "Overview",
-                to: "/",
-              },
-              {
-                label: "Building Chatbots",
-                to: "building-chatbots/creating-a-new-bot",
-              },
-              {
-                label: "Messaging Channels",
-                to: "messaging-channels/supported-channels",
-              },
-              {
-                label: "Going to Production",
-                to: "going-to-production/deploy",
-              },
-              {
-                label: "Chatbot Management",
-                to: "chatbot-management/language-understanding/misunderstood",
-              },
-              {
-                label: "Enterprise",
-                to: "enterprise/licensing/enterprise-licensing",
-              },
-            ],
-          },
-          {
-            title: "Community",
-            items: [
-              {
-                label: "Stack Overflow",
-                href: "https://stackoverflow.com/questions/tagged/botpress",
-              },
-              {
-                label: "Twitter",
-                href: "https://twitter.com/getbotpress",
-              },
-              {
-                label: "Forum",
-                href: "https://github.com/botpress/botpress/discussions",
-              },
-            ],
-          },
-          {
-            title: "More",
-            items: [
-              {
-                label: "GitHub",
-                href: "https://github.com/botpress/botpress",
-              },
-            ],
-          },
-        ],
-        copyright: `Copyright © ${new Date().getFullYear()} Botpress, Inc.`,
+  themeConfig: {
+    announcementBar: {
+      id: 'announcementBar-2', // Increment on change
+      content: `Introducing Botpress 🧠 <a target="_blank" rel="noopener noreferrer" href="https://learn.botpress.com"> Learning Center</a> 🧠 – A New Place to Master Botpress, Chatbots, and Conversational AI`,
+      backgroundColor: '#3276ea',
+      textColor: '#fff',
+    },
+    metadata: [
+      { name: 'robots', content: 'noindex' },
```

Comment:
This overrides the default noIndex config. That's why it was failing. Gotta push globals through the theme config!